src/mmw/apps/modeling/views.py

```diff
@@ -259,6 +175,68 @@ def scenario(request, scen_id):
         return Response(status=status.HTTP_404_NOT_FOUND)
 
 
+@decorators.api_view(['GET', 'POST', 'DELETE'])
+@decorators.permission_classes((IsAuthenticatedOrReadOnly, ))
+def scenario_custom_weather_data(request, scen_id):
```

Comment:
Moved down below `scenario` view instead of editing in place for better organization in the file.